paasta_tools/utils.py

```diff
@@ -3892,9 +3831,7 @@ def is_under_replicated(
 def deploy_blacklist_to_constraints(
```

Comment:
unless we have code that translates constraints into selectors/affinities, these are likely dead code